### ISSUE TITLE
What program are you referring with "/usr/local/bin/resolution"?

### ISSUE THREAD

**Issue by iovis** [Argon-/mpv-config#2]
I see you use it to fetch the resolution but I'd like to know which is (if it's publicly available). Thanks!

**Comment by Argon-** [comment]
Someone asked this once in a comment well hidden inside a commit: https://github.com/Argon-/mpv-config/commit/079bc47a890011836590ec66eda8c9f554f19da1#commitcomment-14879842

It's just a few measly lines of C to query the "real" (that is, unscaled) screen resolution of a OS X machine with an HiDPI display. In such a setup conventional methods return the scaled resolution. As I found no easy way to get the unscaled one I wrote this myself.
I actually put part of the logic inside this program (which was a bad idea), so not the lua script but the `resolution` program compares the resolutions. The return value tells me if the unscaled screen resolution is above or below the threshold.

The reason I'm using this in the first place is that mpv renders on the real screen resolution and certain operations become quickly too taxing for a little iGPU. 60fps with sophisticated scaling is not a problem for a modern iGPU when the resolution is e.g. 1920x1080 but it will be when mpv is in reality rendering at 3840x2400.
